pkg/codegen/main.go

```diff
@@ -7,23 +7,28 @@ import (
 	"github.com/rancher/helm-locker/pkg/crd"
 	"github.com/sirupsen/logrus"
 
-	controllergen "github.com/rancher/wrangler/pkg/controller-gen"
-	"github.com/rancher/wrangler/pkg/controller-gen/args"
+	"github.com/rancher/wrangler/v3/pkg/cleanup"
+	controllergen "github.com/rancher/wrangler/v3/pkg/controller-gen"
+	"github.com/rancher/wrangler/v3/pkg/controller-gen/args"
 )
 
 func main() {
-	if len(os.Args) > 2 && os.Args[1] == "crds" {
-		if len(os.Args) != 3 {
-			logrus.Fatal("usage: ./codegen crds <crd-directory>")
-		}
-		logrus.Infof("Writing CRDs to %s", os.Args[2])
-		if err := crd.WriteFile(os.Args[2]); err != nil {
-			panic(err)
-		}
-		return
+	logrus.Infof("Generating code")
```

Comment:
Why does this need to be pulled out of `pkg/codegen/cleanup`? Not necessarily a blocker just curious

Reply:
It doesn't have to be, but caused issues during the upgrade of dependencies